sky/setup_files/setup.py

```diff
@@ -56,7 +56,9 @@ def parse_footnote(readme: str) -> str:
 
 install_requires = [
     'wheel',
-    'Click',
+    # NOTE: ray 1.13.0 requires click<=8.0.4,>=7.0; our use of shell_complete
+    # requires >= 8.
```

Comment:
update the comment?
```
# NOTE: ray 1.13.0 requires click<=8.0.4,>=7.0. We disable the shell completion for click<8.0 for backward compatibility.
```

very small nit: use ` click<=8.0.4,>=7.0` to be consistent with the comment?

Reply:
agreed, thanks!